Utils.fs

```diff
@@ -338,6 +338,9 @@ let fix f v =
 module Testing =
     open NUnit.Framework
 
+    let assertEqual (a : 'a) (b : 'a) = Assert.AreEqual(a, b)
```

Comment:
ISTR NUnit 3 wants you to write this as `Assert.That(a, Is.EqualTo(b))` or something equally long-winded